src/app/common/has-children.ts

```diff
@@ -1,5 +1,5 @@
 import { ReactNode } from 'react';
 
 export interface HasChildren {
-  children: ReactNode;
+  children?: ReactNode;
```

Comment:
Is this right? Either it has children or not? 🤔

Reply:
`ReactNode` includes `undefined` as a legit node. However without `?` you always have to define the children prop `children={undefined}`. Now you don't.